src/Config/MosaicConfig.ts

```diff
@@ -68,6 +68,8 @@ export class AuxiliaryContracts {
   public ostCoGatewayOrganizationAddress?: Address;
 
   public ostEIP20CogatewayAddress?: Address;
+
```

Comment:
with mosaic-chains now published we should pick #97 and avoid making any more changes to this temporarily cloned file